<launch>
    <node pkg="multiverse_server" type="multiverse_server" name="multiverse_server" output="screen"
        args="7400 7500 7501 7502 7503 7504" />

    <node pkg="multiverse_server" type="multiverse_client.py" name="multiverse_client"
        output="screen" args="7400" />
</launch>